test/e2e/framework/cluster.go

```diff
@@ -94,6 +92,10 @@ func GetClientConfig(cluster *Cluster) clientcmd.ClientConfig {
 		&clientcmd.ClientConfigLoadingRules{ExplicitPath: kubeconfigPath},
 		&clientcmd.ConfigOverrides{
 			CurrentContext: cluster.ClusterName,
+			AuthInfo: api.AuthInfo{
+				Impersonate:       "system",
+				ImpersonateGroups: []string{"system:masters"},
+			},
```

Comment:
why do we need this? Seems that this will affect all the other e2e tests.

Reply:
The rule validates whether the caller is part of the group.  That client currently doesn't belong to any group at all (I checked by printing the results), so all actions that should be allowed became forbidden.